databases/schemas/stored-battles.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE stored_battles (
+	roomid TEXT NOT NULL PRIMARY KEY, -- can store both the num and the formatid
+	input_log TEXT[] NOT NULL,
```

Comment:
Is having an array here preferable to just having newlines in the text? You seem to do `join("\n")` later on?

Reply:
Yeah, I was gonna ask about that. Is there more storage used if it's an array?

Reply:
I'm not sure, probably not too much if so